### ISSUE TITLE
Excalidraw is great but without shortcuts for changing colors it is absolutely unbearable.

### ISSUE THREAD

**Issue by Anos-17** [excalidraw/excalidraw#8064]
Hey, I've been using excalidraw for a while now and the thing is it's such a neat and amazing plugin, I couldn't ever imagining not having it. But this post IS an issue so let's get into that. Excalidraw needs color stroke shortcuts for at least 5 of the layout shortcuts because having to navigate your mouse to the edit button, clicking, navigating to the color strokes, and clicking your desired color stroke doesn't sound that bad but it's very VERY inneficient if you have to change your color every so often. 

I request for there to be a minimum of 5 color shortcuts representing our color palette. This is a to-DIE for feature. This might not be that bad to those who draw using excalidraw but for people who make mind maps with it like me it's very frustrating having to keep doing that over and over again. It's unbearable. I would really appreciate a response on this.

The hot key for it would probably be something like alt+1 and so on or potentially something else. Excalidraw has some really good shortcuts but it's definitely very lackluster when it comes to colors and I think a lot of us feel that way.

Thanks for reading and I hope you implement this feature.
![image](https://github.com/excalidraw/excalidraw/assets/170816583/9c3f8e0f-2528-4100-aced-a4ee63f804b9)

**Comment by dwelle** [comment]
Hi. It was actually planned to add shortcuts to the quick picks but wasn't implemented for some reason.

That said, the way it was going to work was still through first opening a respective color popup using either `s` (stroke color) or `g` (background color) and then using 1-5 to pick from the quick picks.

Having alt-<number> wouldn't work as we wouldn't know whether you want to pick the stroke or the background color.

**Comment by Anos-17** [comment]
> Hi. It was actually planned to add shortcuts to the quick picks but wasn't implemented for some reason.
> 
> That said, the way it was going to work was still through first opening a respective color popup using either `s` (stroke color) or `g` (background color) and then using 1-5 to pick from the quick picks.
> 
> Having alt- wouldn't work as we wouldn't know whether you want to pick the stroke or the background color.

thanks for the reply and, yeah that's a very good idea. Do you know when it might be added?

**Comment by dwelle** [comment]
should will work on it in the few days!

**Comment by TuralAsgar** [comment]
Any news?

**Comment by mghmay** [comment]
+1 for this feature, I find myself opening the palette and then selecting the colour a lot.

**Comment by CamilleHbp** [comment]
> +1 for this feature, I find myself opening the palette and then selecting the colour a lot.

It is already implemented. Pressing `s` shows the stroke colour menu, `g` the background colour menu.

**Comment by NikitaShkaruba** [comment]
+1 for fast color changes in one click

**Comment by Disorrder** [comment]
Yeah, it works great right now. Press S/G, then Q-T, A-G, Z-B to pick from quick palette. I think it's solved

**Comment by secondbrain4-coder** [comment]
maybe you mean Excalidraw independent tool styles or Excalidraw persistent tool properties or Excalidraw global vs per-tool settings or Excalidraw prevent color change across tools or Excalidraw Custom Pen presets guide
I have the same problem
